### ISSUE TITLE
autoClearConsole behaviour

### ISSUE THREAD

**Issue by kharysharpe** [appulate/vscode-file-watcher#11]
Hi @VovanNet / @spostolsky,

What is the expected behaviour of autoClearConsole? 

I am using it to trigger running tests when a file has changed.

settings.json
```
{
    "filewatcher.autoClearConsole": true,
    "filewatcher.commands": [
        {
            "match": "\\.php",
            "isAsync": false,
            "cmd": "cd ${workspaceRoot} && phpunit -v --group tdd",
            "event": "onFileChange"
        }
    ]
}
```

I am expecting it to clear the output before running the command but what is happening now is that it clears immediately after running. Which means I can't read the output of what was just executed.

If this is not the intended behaviour, how can I make it work like above, so I can read the output?

**Comment by listgarten100** [comment]
@kharysharpe Hi! This issue should be fixed in 1.1.7v. Please check it. Good luck)

**Comment by kharysharpe** [comment]
Thank you, I will check it out
